src/renderer/components/kube-object-list-layout/kube-object-list-layout.tsx

```diff
@@ -40,6 +40,10 @@ export interface KubeObjectListLayoutProps<
   store: KubeObjectStore<K, A, D>;
   dependentStores?: SubscribableStore[];
   subscribeStores?: boolean;
+
+  // Customize resource name for e.g. search input ("Search <ResourceName>..."")
```

Comment:
nit: this sort of documentation won't be picked up. Please use `/**` and `*/` instead

Reply:
Fixed